ui/admin/tests/acceptance/auth-methods/create-test.js

```diff
@@ -141,6 +140,76 @@ module('Acceptance | auth-methods | create', function (hooks) {
     assert.strictEqual(authMethod.attributes.api_url_prefix, 'api_url_prefix');
   });
 
+  test('Users can create a new ldap auth method', async function (assert) {
+    assert.expect(23);
+    featuresService.enable('ldap-auth-methods');
+    const authMethodsCount = getAuthMethodsCount();
+    const name = 'ldap auth method';
+    await visit(urls.authMethods);
+
+    await click(NEW_DROPDOWN_SELECTOR);
+    await click(`[href="${urls.newLdapAuthMethod}"]`);
+    await fillIn(NAME_INPUT_SELECTOR, name);
+    await fillIn(DESC_INPUT_SELECTOR, 'description');
+    await fillIn('[name="urls"]', 'url1,url2');
+    await fillIn('[name="certificates"] textarea', 'certificate');
+    await click('[name="certificates"] button');
+    await fillIn('[name="client_certificate"]', 'client cert');
+    await fillIn('[name="client_certificate_key"]', 'client cert key');
+    await click('[name="start_tls"]');
+    await click('[name="insecure_tls"]');
+    await fillIn('[name="bind_dn"]', 'bind dn');
+    await fillIn('[name="bind_password"]', 'password');
+    await fillIn('[name="upn_domain"]', 'upn domain');
+    await click('[name="discover_dn"]');
+    await click('[name="anon_group_search"]');
+    await fillIn('[name="user_dn"]', 'user dn');
+    await fillIn('[name="user_attr"]', 'user attr');
+    await fillIn('[name="user_filter"]', 'user filter');
+    await fillIn('[name="account_attribute_maps"] input', 'attribute');
+    await select('[name="account_attribute_maps"] select', 'email');
+    await click('[name="account_attribute_maps"] button');
+    await fillIn('[name="group_dn"]', 'group dn');
+    await fillIn('[name="group_attr"]', 'group attr');
+    await fillIn('[name="group_filter"]', 'group filter');
+    await click('[name="enable_groups"]');
+    await click('[name="use_token_groups"]');
+    await click(SAVE_BTN_SELECTOR);
+
+    assert.strictEqual(getAuthMethodsCount(), authMethodsCount + 1);
+    const ldapAuthMethod = this.server.schema.authMethods.findBy({ name });
+    assert.strictEqual(ldapAuthMethod.name, name);
+    assert.strictEqual(ldapAuthMethod.description, 'description');
+    assert.deepEqual(ldapAuthMethod.attributes.urls, ['url1', 'url2']);
+    assert.deepEqual(ldapAuthMethod.attributes.certificates, ['certificate']);
+    assert.strictEqual(
+      ldapAuthMethod.attributes.client_certificate,
+      'client cert'
+    );
+    assert.strictEqual(
+      ldapAuthMethod.attributes.client_certificate_key,
+      'client cert key'
```

Comment:
@lisbet-alvarez we are deleting the `client_certificate_key` in this PR, #1786

Reply:
Thank you! I will also go check out that PR.